site/troubleshooting.md

```diff
@@ -142,7 +142,7 @@ is waiting for a while before connecting again.
 There may also be further sections for the
 [IP address allocator](ipam.html#troubleshooting),
 [weaveDNS](weavedns.html#troubleshooting), and
-[proxy](proxy.html#troubleshooting).
+[Weave Docker API Proxy](proxy.html#troubleshooting).
```

Comment:
I'm calling it the "*Weave* Docker API Proxy" here because I reckon there isn't enough context here to tell that this is a weave component and not a docker component.